app/services/course/assessment/question/programming/python/python_package_service.rb

```diff
@@ -151,7 +151,7 @@ def generate_zip_file(data_files_to_keep)
       index = 1
       [:public, :private, :evaluation].each do |test_type|
         tests[test_type].try(:each) do |test|
-          test_fn = "    def test_#{test_type}_#{index}(self):\n"\
+          test_fn = "    def test_#{test_type}_#{format('%02i', index)}(self):\n"\
```

Comment:
Oh, just change this (and the next few lines) into a multi-line string only. It should look like that: 
```ruby
test_fn = <<-PYTHON
  def text_#{test_type}_#(format('...)
    ....
PYTHON
```